stock_scanner_inventory/data/Inventory/scanner_scenario_step_inventory_location_selection.py

```diff
@@ -0,0 +1,18 @@
+# flake8: noqa
+# Use <m> or <message> to retrieve the data transmitted by the scanner.
+# Use <t> or <terminal> to retrieve the running terminal browse record.
+# Put the returned action code in <act>, as a single character.
+# Put the returned result or message in <res>, as a list of strings.
+# Put the returned value in <val>, as an integer
+
+terminal.write({'tmp_val3': message})
```

Comment:
All "undefined" variables are available at runtime, because these files are loaded into database (in the "code" field of the "scanner.scenario.step" model).
That's why I put the "flake8: noqa" comment on top of each scenario's python files.